src/main/java/appeng/parts/p2p/PartP2PTunnel.java

```diff
@@ -308,17 +308,17 @@ public boolean onPartShiftActivate( final EntityPlayer player, final EnumHand ha
 			final IMemoryCard mc = (IMemoryCard) is.getItem();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Method appeng.parts.p2p.PartP2PTunnel.onPartShiftActivate(EntityPlayer, EnumHand, Vec3d) appears to call the same method on the same object redundantly [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=fb-contrib%3APRMC_POSSIBLY_REDUNDANT_METHOD_CALLS)